.devcontainer/cuda11.8-conda/devcontainer.json

```diff
@@ -8,6 +8,11 @@
       "BASE": "rapidsai/devcontainers:24.06-cpp-cuda11.8-mambaforge-ubuntu22.04"
     }
   },
+  "runArgs": [
+    "--rm",
+    "--name",
+    "rapids-${localWorkspaceFolderBasename}-24.06-cuda11.8-conda"
```

Comment:
will the 24.06 version here be kept up to date w/ the update-version.sh script? 

https://github.com/rapidsai/cuvs/blob/aea7f5423fb78db2258442c208e2641298b6f399/ci/release/update-version.sh#L89-L94

Reply:
Yep, updated